data/src/main/java/com/architect/coders/mu8/data/events/EventsResponse.kt

```diff
@@ -0,0 +1,11 @@
+package com.architect.coders.mu8.data.events
+
+import com.architect.coders.mu8.data.response.common.ThumbnailResponse
+import com.architect.codes.mu8.utils.EMPTY_STRING
+
+data class EventsResponse(
+    val id: Long = 0L,
+    val title: String = EMPTY_STRING,
+    val description: String = EMPTY_STRING,
+    val thumbnail: ThumbnailResponse
```

Comment:
put the default value for this as en empty object, `ThumbnailResponse` already have default values to avoid nullability.

Reply:
Done